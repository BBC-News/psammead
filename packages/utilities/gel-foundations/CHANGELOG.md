# Gel Foundations Changelog

| Version | Description |
|---------|-------------|
| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
| 0.1.1   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
| 0.1.0   | [PR#221](https://github.com/BBC-News/psammead/pull/221) Create initial package, pulled in from gel-constants and gel-foundations-styled-components packages. |

