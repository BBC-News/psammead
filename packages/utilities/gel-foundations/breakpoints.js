module.exports = require('./dist/breakpoints'); // eslint-disable-line import/no-unresolved
