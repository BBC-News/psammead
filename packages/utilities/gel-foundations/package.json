{
  "name": "drew-testing-123",
  "version": "1.0.6",
  "main": "dist/index.cjs.js",
  "module": "dist/index.es.js",
  "description": "A range of string constants for use in CSS, intended to help implement BBC GEL-compliant webpages and components.",
  "repository": {
    "type": "git",
    "url": "https://github.com/bbc/psammead/tree/latest/packages/utilities/gel-foundations"
  },
  "author": {
    "name": "Psammead Maintainers",
    "email": "PsammeadMaintainers@bbc.co.uk"
  },
  "files": [
    "dist/"
  ],
  "license": "Apache-2.0",
  "bugs": {
    "url": "https://github.com/bbc/psammead/issues"
  },
  "homepage": "https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/README.md",
  "keywords": [
    "bbc",
    "utilities",
    "gel",
    "breakpoints",
    "grid",
    "spacing",
    "typography"
  ],
  "devDependencies": {
    "@bbc/psammead-test-helpers": "^1.0.1"
  }
}
