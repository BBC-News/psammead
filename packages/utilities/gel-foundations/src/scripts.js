export latin from './scripts/latin-and-cyrillic';
export cyrillic from './scripts/latin-and-cyrillic';
export latinDiacritics from './scripts/latin-with-diacritics';
export arabic from './scripts/arabic';
export arabicPashto from './scripts/arabic-pashto';
export bengali from './scripts/bengali';
export burmese from './scripts/burmese';
export hindi from './scripts/devanagari-and-gurmukhi';
export nepali from './scripts/devanagari-and-gurmukhi';
export sinhalese from './scripts/sinhalese';
export tamil from './scripts/tamil';
export thai from './scripts/thai';
export chinese from './scripts/no-ascenders-or-descenders';
export korean from './scripts/no-ascenders-or-descenders';
export ethiopic from './scripts/no-ascenders-or-descenders';
