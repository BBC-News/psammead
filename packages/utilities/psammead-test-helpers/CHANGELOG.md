# Psammead Test Helpers Changelog

| Version | Description |
|---------|-------------|
| 0.3.0   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Adds a helper method for testing package export values and uses it to test its own exports. |
| 0.2.1   | [PR#255](https://github.com/BBC-News/psammead/pull/255) Bumps package version to fix what appears to be an erroneously-published version. |
| 0.1.2   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
| 0.1.1   | [PR#173](https://github.com/BBC-News/psammead/pull/173) Update PRs welcome link |
| 0.1.0   | [PR#80](https://github.com/BBC-News/psammead/pull/80) Create initial package, pulled in from [simorgh](https://github.com/BBC-News/simorgh). |
