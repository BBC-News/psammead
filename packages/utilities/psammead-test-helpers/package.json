{
  "name": "@bbc/psammead-test-helpers",
  "version": "0.3.0",
  "main": "dist/index.js",
  "description": "A collection of helper methods for implementing Jest snapshot tests for styled-components, required by many Psammead components.",
  "repository": {
    "type": "git",
    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/utilities/psammead-test-helpers"
  },
  "author": {
    "name": "Psammead Maintainers",
    "email": "PsammeadMaintainers@bbc.co.uk"
  },
  "license": "Apache-2.0",
  "bugs": {
    "url": "https://github.com/bbc/psammead/issues"
  },
  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/utilities/psammead-test-helpers/README.md",
  "keywords": [
    "bbc",
    "utilities",
    "test",
    "helpers",
    "jest",
    "snapshot"
  ],
  "dependencies": {
    "jest-styled-components": "^6.3.1",
    "react-test-renderer": "^16.6.3"
  }
}
