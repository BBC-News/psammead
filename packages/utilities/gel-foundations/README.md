# gel-foundations &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-foundations.svg)](https://www.npmjs.com/package/@bbc/gel-foundations) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)

This package provides a range of string constants for use in CSS, intended to help implement [BBC GEL-compliant](https://www.bbc.co.uk/gel/articles/what-is-gel) webpages and components.

## Exports

`/breakpoints` - GEL breakpoints, as well as typography breakpoints. These use the GEL grid sizes which can be found in the [GEL Grid guidelines](https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes).  
`/spacings` - GEL spacings and GEL Grid margins and gutters. These use the GEL grid spacing sizes which can be found in the [GEL Grid guidelines](https://www.bbc.co.uk/gel/guidelines/grid#spacing-layout).  
`/typography` - GEL typography. These are based on the gel typography standard which can be found in the [GEL guidelines](https://www.bbc.co.uk/gel/guidelines/typography). [More details on how these sizes were implemented for this package are available here.](./typography_sizes_web.md)

## Installation

```jsx
npm install @bbc/gel-foundations --save
```

## Usage

```jsx
import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';

import { GEL_GUTTER_ABOVE_600PX } from '@bbc/gel-foundations/spacings';

import { GEL_BREVIER } from '@bbc/gel-foundations/typography';
```

These values can then be used directly within CSS declarations in code:

```jsx
import { css } from 'styled-components';

const SomeStyledComponent = css`
  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
    grid-gap: ${GEL_GUTTER_ABOVE_600PX};
    ${GEL_BREVIER};
  }
`;
```

To allow the typography to be fully accessible and responsive, please note that you should apply a default font-size to the document root (e.g. `html { font-size: 100% }`).

Our typography uses `em` for font-size and `rem` for line-height. `em` allows modularity of components: you can change the component font-size by changing the font-size of its container. `rem` is relative to the document root, so we use that for line-height and spacing for a consistent look-and-feel across the document. You can read our [detailed analysis of "REMs vs EMs for spacing"](https://github.com/bbc/simorgh/blob/latest/docs/Spacing-Units.md) for more information.

## Contributing

When **adding** a new export to this utility package the [export tests](https://github.com/bbc/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/gel-foundations/index.test.jsx#L13-L59) also need to be updated. When **removing** an exisiting export from this utility package the [export tests](https://github.com/bbc/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/gel-foundations/index.test.jsx#L13-L59) need to be updated and the package version requires a major change (EG: 1.2.1 -> 2.0.0) as this would be considered a breaking change as functionality is being removed.

Psammead is completely open source. We are grateful for any contributions, whether they be new components, bug fixes or general improvements. Please see our primary contributing guide which can be found at [the root of the Psammead respository](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md).

### [Code of Conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)

We welcome feedback and help on this work. By participating in this project, you agree to abide by the [code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md). Please take a moment to read it.

### License

Psammead is [Apache 2.0 licensed](https://github.com/bbc/psammead/blob/latest/LICENSE).
