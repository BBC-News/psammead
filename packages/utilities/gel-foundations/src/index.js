import * as breakpoints from './breakpoints';
import * as propTypes from './prop-types';
import * as scripts from './scripts';
import * as spacings from './spacings';
import * as typography from './typography';

export { breakpoints };
export { propTypes };
export { scripts };
export { spacings };
export { typography };
