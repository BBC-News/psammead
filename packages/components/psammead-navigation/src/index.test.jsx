import React from 'react';
import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
import { render } from '@testing-library/react';
import { latin } from '@bbc/gel-foundations/scripts';
import {
  CanonicalScrollableNavigation,
  AmpScrollableNavigation,
} from './ScrollableNavigation';
import Navigation, { NavigationUl, NavigationLi } from './index';
import igboNavData from '../testHelpers/igbo';

const NavigationExample = (
  <Navigation script={latin} skipLinkText="Wụga n’ọdịnaya" service="news">
    <NavigationUl>
      {igboNavData.map((item, index) => {
        const { title, url } = item;
        const active = index === 0;

        return (
          <NavigationLi
            key={title}
            url={url}
            script={latin}
            active={active}
            currentPageText="Current page"
            service="news"
            data-navigation="test_navigation"
          >
            {title}
          </NavigationLi>
        );
      })}
    </NavigationUl>
  </Navigation>
);

describe('Navigation', () => {
  shouldMatchSnapshot('should render correctly', NavigationExample);
});

describe('Scrollable Navigation', () => {
  window.matchMedia = jest.fn().mockImplementation(query => {
    return {
      matches: true,
      media: query,
      addListener: jest.fn(),
      removeListener: jest.fn(),
    };
  });

  shouldMatchSnapshot(
    'should render Canonical version correctly',
    <CanonicalScrollableNavigation>
      {NavigationExample}
    </CanonicalScrollableNavigation>,
  );

  shouldMatchSnapshot(
    'should render AMP version correctly',
    <AmpScrollableNavigation>{NavigationExample};</AmpScrollableNavigation>,
  );
});

describe('Assertions', () => {
  it('should add extra props passed to the NavigationLi link', () => {
    const { container } = render(
      <NavigationLi
        key="test-key"
        url="http://test.url"
        script={latin}
        currentPageText="Current page"
        service="news"
        active
        data-navigation="test_navigation"
      >
        Testing exta props
      </NavigationLi>,
    );
    expect(
      container.querySelector('a').getAttribute('data-navigation'),
    ).toEqual('test_navigation');
  });
});
