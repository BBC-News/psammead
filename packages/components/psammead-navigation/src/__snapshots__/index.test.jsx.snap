// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render correctly 1`] = `
.c10 {
  -webkit-clip-path: inset(100%);
  clip-path: inset(100%);
  -webkit-clip: rect(1px,1px,1px,1px);
  clip: rect(1px,1px,1px,1px);
  height: 1px;
  overflow: hidden;
  position: absolute;
  width: 1px;
  margin: 0;
}

.c1 {
  position: relative;
  max-width: 80rem;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-right: 0;
}

.c2 {
  position: absolute;
  -webkit-clip-path: inset(100%);
  clip-path: inset(100%);
  -webkit-clip: rect(1px,1px,1px,1px);
  clip: rect(1px,1px,1px,1px);
  height: 1px;
  width: 1px;
  overflow: hidden;
  padding: 0.75rem 0.5rem;
  background-color: #ffffff;
  border: 0.1875rem solid #000;
  color: #333;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 0.9375rem;
  line-height: 1.25rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
}

.c2:focus {
  -webkit-clip-path: none;
  clip-path: none;
  -webkit-clip: auto;
  clip: auto;
  height: auto;
  width: auto;
  top: -3rem;
  left: 0.5rem;
}

.c5 {
  list-style-type: none;
  padding: 0;
  margin: 0;
  position: relative;
  overflow: hidden;
  overflow-x: scroll;
  white-space: nowrap;
}

.c5::-webkit-scrollbar {
  display: none;
}

.c13 {
  list-style-type: none;
  padding: 0;
  margin: 0;
  position: relative;
  overflow: hidden;
}

.c7 {
  display: inline-block;
  position: relative;
  z-index: 2;
}

.c14 {
  display: block;
  position: relative;
  z-index: 2;
}

.c4 {
  font-size: 0.9375rem;
  line-height: 1.25rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
  color: #FDFDFD;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
  padding: 0.75rem 1rem;
  white-space: nowrap;
}

.c4:hover::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
}

.c4:focus::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
  top: 0;
  border: 0.25rem solid #FFFFFF;
}

.c8 {
  font-size: 0.9375rem;
  line-height: 1.25rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
  color: #FDFDFD;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
  padding: 0.75rem 1rem;
  white-space: nowrap;
}

.c8:hover::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
  border-bottom: 0.3125rem solid #FFFFFF;
}

.c8:focus::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
  top: 0;
  border: 0.25rem solid #FFFFFF;
}

.c11 {
  color: #FDFDFD;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
  padding: 0.75rem 1rem;
  white-space: nowrap;
}

.c11:hover::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
}

.c11:focus::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
  top: 0;
  border: 0.25rem solid #FFFFFF;
}

.c15 {
  font-size: 0.9375rem;
  line-height: 1.25rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
  color: #B80000;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
  padding: 0.75rem 1rem;
}

.c15:hover::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #B80000;
  border-bottom: 0.3125rem solid #B80000;
}

.c15:focus::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #B80000;
  top: 0;
  border: 0.25rem solid #FFFFFF;
}

.c17 {
  font-size: 0.9375rem;
  line-height: 1.25rem;
  font-family: ReithSans,Helvetica,Arial,sans-serif;
  font-weight: 400;
  font-style: normal;
  color: #B80000;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
  padding: 0.75rem 1rem;
}

.c17:hover::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #B80000;
}

.c17:focus::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #B80000;
  top: 0;
  border: 0.25rem solid #FFFFFF;
}

.c9::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #FFFFFF;
}

.c16::after {
  content: '';
  position: absolute;
  left: 0;
  right: 0;
  bottom: 0;
  border-bottom: 0.25rem solid #B80000;
}

.c0 {
  background-color: #B80000;
}

.c0 .c6::after {
  left: 0;
}

.c12 {
  max-width: 80rem;
  overflow: scroll;
  border-bottom: solid #B80000;
  display: none;
  background-color: #FFFFFF;
  margin: 0 auto;
  position: absolute;
  left: 0;
  border-left: solid #B80000;
  border-right: solid #B80000;
  z-index: 10;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  pointer-events: auto;
  max-height: 85vh;
}

.c3 {
  position: relative;
  border-right: #FFFFFF solid;
}

@media (min-width:20rem) and (max-width:37.4375rem) {
  .c2 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width:37.5rem) {
  .c2 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width:37.5rem) {
  .c2:focus {
    top: -4rem;
  }
}

@media (min-width:80rem) {
  .c2:focus {
    left: 0;
  }
}

@media (min-width:20rem) and (max-width:37.4375rem) {
  .c4 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width:37.5rem) {
  .c4 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (max-width:24.9375rem) {
  .c4 {
    padding: 0.75rem 0.5rem;
  }
}

@media (min-width:20rem) and (max-width:37.4375rem) {
  .c8 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width:37.5rem) {
  .c8 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (max-width:24.9375rem) {
  .c8 {
    padding: 0.75rem 0.5rem;
  }
}

@media (max-width:24.9375rem) {
  .c11 {
    padding: 0.75rem 0.5rem;
  }
}

@media (min-width:20rem) and (max-width:37.4375rem) {
  .c15 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width:37.5rem) {
  .c15 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (max-width:24.9375rem) {
  .c15 {
    padding: 0.75rem 0.5rem;
  }
}

@media (min-width:20rem) and (max-width:37.4375rem) {
  .c17 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (min-width:37.5rem) {
  .c17 {
    font-size: 1rem;
    line-height: 1.25rem;
  }
}

@media (max-width:24.9375rem) {
  .c17 {
    padding: 0.75rem 0.5rem;
  }
}

<div>
  <nav
    class="c0"
    dir="ltr"
    role="navigation"
  >
    <div
      class="c1"
    >
      <a
        class="c2"
        href="#content"
      >
        Wụga n’ọdịnaya
      </a>
      <div
        class="c3"
        dir="ltr"
      >
        <button
          style="border: 0px; padding: 0px; background-color: inherit;"
          type="button"
        >
          <a
            class="c4"
          >
            Menu
            <svg
              fill="white"
              focusable="false"
              height="8px"
              style="margin-left: 8px;"
              viewBox="0 0 32 32"
              width="8px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                down
              </title>
              <path
                d="M16 29L32 3h-7.2L16 18.3 7.2 3H0l16 26z"
              />
            </svg>
          </a>
        </button>
      </div>
      <ul
        class="c5"
        role="list"
      >
        <li
          class="c6 c7"
          role="listitem"
        >
          <a
            class="c8"
            href="/igbo"
          >
            <span
              class="c9"
              role="text"
            >
              <span
                class="c10"
              >
                Current page
                , 
              </span>
              Akụkọ
            </span>
          </a>
        </li>
        <li
          class="c6 c7"
          role="listitem"
        >
          <a
            class="c4"
            href="/igbo/egwuregwu"
          >
            Egwuregwu
          </a>
        </li>
        <li
          class="c6 c7"
          role="listitem"
        >
          <a
            class="c4"
            href="/igbo/media/video"
          >
            Ihe nkiri
          </a>
        </li>
        <li
          class="c6 c7"
          role="listitem"
        >
          <a
            class="c4"
            href="/igbo/popular/read"
          >
            Nke ka ewuewu
          </a>
        </li>
        <a
          class="c11"
        >
          <div
            style="width: 1rem; height: 1px;"
          />
        </a>
      </ul>
      <div
        style="width: 2rem; height: 100%; flex-shrink: 0; z-index: 3; overflow: hidden; position: absolute; right: 0px; top: 0px; pointer-events: none;"
      />
    </div>
  </nav>
  <div
    class="c12"
    dir="ltr"
  >
    <ul
      class="c13"
      role="list"
    >
      <li
        class="c6 c14"
        role="listitem"
      >
        <a
          class="c15"
          href="/igbo"
        >
          <span
            class="c16"
            role="text"
          >
            <span
              class="c10"
            >
              Current page
              , 
            </span>
            Akụkọ
          </span>
        </a>
      </li>
      <li
        class="c6 c14"
        role="listitem"
      >
        <a
          class="c17"
          href="/igbo/egwuregwu"
        >
          Egwuregwu
        </a>
      </li>
      <li
        class="c6 c14"
        role="listitem"
      >
        <a
          class="c17"
          href="/igbo/media/video"
        >
          Ihe nkiri
        </a>
      </li>
      <li
        class="c6 c14"
        role="listitem"
      >
        <a
          class="c17"
          href="/igbo/popular/read"
        >
          Nke ka ewuewu
        </a>
      </li>
    </ul>
  </div>
</div>
`;
